cmd/auth/main.go

```diff
@@ -128,19 +129,18 @@ func main() {
 
 	svc := newService(db, dbTracer, cfg.secret, logger, readerConn, writerConn, cfg.loginDuration)
 
+	hs := httpserver.New(ctx, cancel, serviceName, defListenAddress, cfg.httpPort, httpapi.MakeHandler(svc, tracer, logger), cfg.serverCert, cfg.serverKey, logger)
+	gs := grpcserver.New(ctx, cancel, serviceName, defListenAddress, cfg.httpPort, grpcapi.NewServer(tracer, svc), cfg.serverCert, cfg.serverKey, logger)
```

Comment:
are both HTTP and GRPC listening to the same port? 
this would be a nice thing to have maybe we can follow this pattern https://github.com/grpc-ecosystem/grpc-gateway